src/connectors/uwu-radio.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+// If I understood correctly, these are all just backups in case the Media
+// Session API isn't supported.
+Connector.useMediaSessionApi();
```

Comment:
This is more for using the media session API for getting meta data about the playing music

Reply:
I made this connector before I added a polyfill for the Media Session API so it also works on older brothers, removed the other selectors now.